cluster/ovm-pvc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: OfflineVirtualMachine
+metadata:
+  name: my-vm
+  labels:
+    kubevirt.io/my: label
+spec:
+  running: false 
+  template:
+    metadata:
+      labels:
+        kubevirt.io/my: label
+    spec:
+      domain:
+        resources:
+          requests:
+            memory: 64M
+        devices:
+          disks:
+            - name: pvcvolume
+              volumeName: pvcvolume
+              disk:
+                bus: virtio
+            - name: pvcdata
+              volumeName: pvcdata
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              volumeName: cloudinitvolume
+              disk:
+                bus: virtio
+      volumes:
+        - name: pvcvolume
+          persistentVolumeClaim:
+            claimName: disk-alpine
+        - name: pvcdata
+          persistentVolumeClaim:
+            claimName: disk-custom 
+        - name: cloudinitvolume
```

Comment:
I don't believe our alpine image supports cloud-init.  This won't hurt anything, but it doesn't necessarily solve anything either.